### PR TITLE
Added  Controller(initial) as it will unlock other stories

### PR DESCRIPTION
**Pull request by nitinprabhu**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RPE-218

### Change description ###

- Added initial controller as it will unlock publish spec and s2s stories
- Not added test cases as the exact inputs/outputs are not known at the moment.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/sendletter/controllers/SendLetterController.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.sendletter.model.Letter;
+import uk.gov.hmcts.reform.sendletter.services.LetterService;
+
+import java.util.Map;
+
+@RestController
+@RequestMapping(
+    path = "letter",
```
Comment (rkondratowicz) [inline src/main/java/uk/gov/hmcts/reform/sendletter/controllers/SendLetterController.java]: `/letters`
https://hmcts.github.io/restful-api-standards/#134

Comment (nitinprabhu) [reply src/main/java/uk/gov/hmcts/reform/sendletter/controllers/SendLetterController.java]: Thanks Rafal.Will update this :)

File: src/main/java/uk/gov/hmcts/reform/sendletter/controllers/SendLetterController.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    public SendLetterController(LetterService letterService) {
+        this.letterService = letterService;
+    }
+
+    @PostMapping(consumes = {MediaType.APPLICATION_JSON_VALUE})
+    @ApiOperation(value = "Send letter to print and post service")
+    @ApiResponses({
+        @ApiResponse(code = 200, message = "Successfully sent letter"),
+    })
+    public void sendLetter(@RequestBody Map<String, String> addressDetails) {
```
Comment (rkondratowicz) [inline src/main/java/uk/gov/hmcts/reform/sendletter/controllers/SendLetterController.java]: is void 200?

Comment (nitinprabhu) [reply src/main/java/uk/gov/hmcts/reform/sendletter/controllers/SendLetterController.java]: No that will change.Currently kept it void but will update it later

**Pull request merged**